app/views/reviews/index.html.slim

```diff
@@ -2,45 +2,97 @@
 - content_for :page_heading, "My Reviews"
 
 - if @reviews.size > 0
+  .row
+    .col-xs-6.myreviews-stats-col1
+      .container-fluid#total-reviews-container
+        h3 Total Reviews Written
+        h1#myreviews-stats-number =@reviews.count
+    .col-xs-6.myreviews-stats-col2
+      .container-fluid#average-professor-ratings-container
+        h3 Average Professor Ratings
+        h1#myreviews-stats-number =@avg_prof_rating
+  .container-fluid#addreview-container = link_to new_review_path
+    .addreview
+      .row
+        .col-xs-12
+          h4#addreview-label Write Another Review
+        .col-xs-12
+          span.glyphicon.glyphicon-plus
+
+  br
   - @semesters.each do |semester|
     .row
       .col-xs-12
-        h3= "#{semester.season} #{semester.year}"
+        h2#semester-header = "#{semester.season} #{semester.year}"
     - @reviews_map[semester.number].each do |review|
-      .row
-        .col-xs-2
-          = link_to "#{review.course.mnemonic_number}", review.course
-        .col-xs-3
-          = link_to review.professor.full_name, review.professor
-        .col-xs-2
-          = link_to "Edit Review", edit_review_path(review)
-        .col-xs-5
-          a href="#{course_path(review.course_id, :p => review.professor_id)}"
-            span Go To Class 
-            span.glyphicon.glyphicon-arrow-right
+      div.myreview-header
+        .row.myreview-header-row
+          .col-xs-2
+            h3= link_to "#{review.course.mnemonic_number}", review.course
+          .col-xs-2
+            = link_to review.professor.full_name, review.professor
+          .col-xs-3
+            a href="#{course_path(review.course_id, :p => review.professor_id)}"
+              span Go To Class 
+              span.glyphicon.glyphicon-arrow-right
+          .col-xs-5
+            div.edit-review
+              = link_to "Edit Review", edit_review_path(review)
+      .container-fluid.myreview-container
+        .row
+          .col-xs-12
+            - if review.comment == ""
+              .comment <i>No comment for this review, edit to write a comment!</i>
+            - else
+              .comment= h(review.comment).gsub(/\n/, '<br/>').html_safe
```

Comment:
Honestly, not sure why it's there but reviews are displayed like that on the courses show.html and I wanted to keep it consistent. Seems like it's being displayed the same with the line above or just as 'review.comment' though.